test/connection_test.exs

```diff
@@ -5,20 +5,7 @@ defmodule RigInboundGateway.ConnectionTest do
 
   alias HTTPoison
 
-  @dispatch Confex.fetch_env!(:rig, RigInboundGatewayWeb.Endpoint)[:https][:dispatch]
-  @port 47_210
-
-  setup_all do
-    dispatch = :cowboy_router.compile(@dispatch)
-    server_name = __MODULE__
-    {:ok, _pid} = :cowboy.start_clear(server_name, [port: @port], %{env: %{dispatch: dispatch}})
-
-    on_exit(fn ->
-      :cowboy.stop_listener(server_name)
-    end)
-
-    :ok
-  end
```

Comment:
did you remove this by accident? if not, I'd be curious as to why this is no longer needed

Reply:
This caused **a lot** of problems when running locally. I removed it and it worked. I didn't mean to commit this but ever since, I didn't get any further problems with this test being flaky. When explicitly running this test (`mix test test/connection_test.exs`) I also don't have any problems (when compared to before). But since I don't think that me randomly deleting code is a fix (and I don't really know why this is "fixed"), I will add this back.